pyshelf/search/update_manager.py

```diff
@@ -1,9 +1,46 @@
 from pyshelf.search.metadata import Metadata
+from elasticsearch_dsl.query import Q
+from elasticsearch_dsl import Search
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import scan, bulk
 
 
 class UpdateManager(object):
-    def __init__(self, logger):
+    def __init__(self, logger, host, index):
         self.logger = logger
+        self.connection = Elasticsearch(host)
+        self.index = index
+
+    def remove_unlisted_documents(self, ex_key_list):
```

Comment:
Is ex supposed to be for existing?  Can you just type existing_key_list.  I don't mind shortened variable names for stuff internal to functions but I would like well described parameters and class level variables.